torchao/csrc/metal/benchmarks/llama_bench.mm

```diff
@@ -0,0 +1,407 @@
+#include <Metal/Metal.h>
```

Comment:
Interesting that the structure is written in objectC. Does it include all ops or just linear? Asking because I don't find softmax, rope, rsnorm, etc. 

Would be good to add comments on what this file is for.

Reply:
That is WIP since there are no readily available metal kernels plus thats not where most of the time is spent anyway